src/hatti/views/map.cljs

```diff
@@ -329,6 +458,49 @@
                    :on-click (click-fn #(om/update! geofield field))}]
                  (get-label field) [:br]])]]]))))))
 
+(defn map-layer-selector
+  [cursor owner]
+  (reify
+    om/IInitState
+    (init-state [_]
+      {:expanded false
+       :styles ["basic" "streets" "bright" "light" "dark" "satellite"]
```

Comment:
https://www.mapbox.com/android-sdk/examples/default-styles/

Note: we want to use: satellite-streets-v9 not just satellite-v9.
Bright doesn't exist anymore but we'll want to add outdoors.



On Tue, Aug 16, 2016 at 7:54 AM, Roy Rutto <notifications@github.com> wrote:

> In src/hatti/views/map.cljs
> <https://github.com/onaio/hatti/pull/146#discussion_r74928188>:
>
> > @@ -329,6 +458,49 @@
> >                     :on-click (click-fn #(om/update! geofield field))}]
> >                   (get-label field) [:br]])]]]))))))
> >
> > +(defn map-layer-selector
> > +  [cursor owner]
> > +  (reify
> > +    om/IInitState
> > +    (init-state [_]
> > +      {:expanded false
> > +       :styles ["basic" "streets" "bright" "light" "dark" "satellite"]
>
> Yes, if we build our custom map styles on Mapbox, then they'll need to be
> added here and the style link (https://github.com/onaio/
> hatti/pull/146/files/70df3bdf47fffad7dd1d13a627cc8227eeb608ae#diff-
> 765a06264ee0391b4aede90c1206b186R496) will change slightly. But these are
> the defaults offered by Mapbox.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/onaio/hatti/pull/146/files/70df3bdf47fffad7dd1d13a627cc8227eeb608ae#r74928188>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADiQS0tvInNrorxebuVPclXO9i0le0Nks5qgbMegaJpZM4Jkf22>
> .
>